files/qsub.pl

```diff
@@ -540,7 +540,19 @@ sub parse_script
     # replace PBS_JOBID in -o / -e
     #   torque sets stdout/err in cwd -> so force abspath like done above
     # All script changes here
+    my $change_line = sub {
+        my ($line, $env_var, $prefix) = @_;
+        if ($line =~ m/^\s*#PBS.*?\$\{?$prefix($env_var)\}?/ && $ENV{$1}) {
```

Comment:
What if there are multiple environment variables on a single line `#PBS` line?

Reply:
@boegel I would like to see an example. (I will fix it anyway)

Reply:
(and actually the current script changes all the env_vars (see the `g` modifier after the last `/`)

Reply:
^in the next line

Reply:
OK, then just add a test case for it. Example:

```
#PBS -o $VSC_DATA/$VSC_INSTITUTE_CLUSTER/results